pkg/plugin/v3/scaffolds/internal/templates/config/controller/controller.go

```diff
@@ -76,6 +76,13 @@ type {{ .Resource.Kind }}Reconciler struct {
 // +kubebuilder:rbac:groups={{ .Resource.Domain }},resources={{ .Resource.Plural }},verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups={{ .Resource.Domain }},resources={{ .Resource.Plural }}/status,verbs=get;update;patch
 
+// Reconcile reads that state of the cluster for a {{ .Resource.Kind }} object 
+// and makes changes based on the state and what is in the {{ .Resource.Kind }}.Spec.
+// TODO(user): Modify this Reconcile function to implement your Controller logic.
```

Comment:
I am not sure if we can say that Reconcile read the state of the cluster. The reconcile func will be trigged in many scenarios to reconconcile the data of an object. See the docs: https://github.com/kubernetes-sigs/controller-runtime/blob/v0.6.3/pkg/reconcile/reconcile.go#L88.

Reply:
Updated the comments. Let me know what you think...